### PR TITLE
[DOC] API tidy

### PR DESCRIPTION
**Pull request by TonyBagnall**
Update and tidy up the API a bit

### CODE REVIEW

File: docs/api_reference.md [hunk 1]
```diff
@@ -28,5 +28,5 @@ api_reference/segmentation
 api_reference/similarity_search
 api_reference/transformations
 api_reference/utils
-api_reference/file_specifications/ts
+api_reference/visualization
```
Comment (dguijo) [inline docs/api_reference.md:31]: The module is named `visualisation`, using UK spelling. Not sure if api should keep the same nomenclature though.

**Review by dguijo — CHANGES_REQUESTED** [review]
Just a minor change, Tony. Thank you again!

File: docs/api_reference.md (hunk 1 above)
Comment (TonyBagnall) [reply docs/api_reference.md:31]: imagine me getting that wrong

**Review by dguijo — APPROVED** [review]
LGTM Thank you Tony!

**Pull request merged**